Gems/Multiplayer/Code/Tests/MultiplayerComponentTests.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <CommonNetworkEntitySetup.h>
+#include <MockInterfaces.h>
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzCore/UnitTest/UnitTest.h>
+#include <AzNetworking/Serialization/StringifySerializer.h>
+#include <AzTest/AzTest.h>
+#include <Multiplayer/Components/MultiplayerComponent.h>
+
+namespace Multiplayer
+{
+    using namespace testing;
+    using namespace ::UnitTest;
```

Comment:
We should avoid using namespace at namespace scope as well.

Reply:
Whoops. Accidentally cut-and-pasted this from a different test file. Removed it from both mine and the other file. Turns out they weren't needed in either one, making them even *more* egregious. :(